zgw_consumers/test/factories.py

```diff
@@ -0,0 +1,38 @@
+import factory
+
+from zgw_consumers.models import Service
+
+
+class UriPathFaker(factory.Faker):
+    def __init__(self, **kwargs):
+        super().__init__("uri_path", **kwargs)
+
+    def generate(self, extra_kwargs=None):
+        uri_path = super().generate(extra_kwargs)
+        # faker generates them without trailing slash, but let's make sure this stays true
+        # zgw_consumers.Service normalizes api_root to append missing trailing slashes
+        assert not uri_path.endswith("/")
+        return f"{uri_path}/"
+
+
+class ServiceFactory(factory.django.DjangoModelFactory):
+    label = factory.Sequence(lambda n: f"API-{n}")
+    api_root = UriPathFaker()  # FIXME: this should be a fully qualified URL
+
+    class Meta:
+        model = Service
+        django_get_or_create = ("api_root",)
+
+    class Params:
+        with_server_cert = factory.Trait(
+            server_certificate=factory.SubFactory(
+                "simple_certmanager_ext.tests.factories.CertificateFactory",
```

Comment:
this also depends on a new simple-cert manager version that ships the factories before we can merge this, and then the minimum version of simple-certmanager will need to be set in `setup.cfg`

Reply:
Yes just saw that when trying to add tests from open-forms, will first tackle simple cert manager